docs/eventing/event_registry.md

```diff
@@ -0,0 +1,260 @@
+# Event Registry
+
```

Comment:
Update the title of this file to be "event-registry.md" ( changing the underscore to a hyphen) for consistency with the other files in the docs directory.

Reply:
Done, thanks!